ath79: Add GL.iNet AR150 LED triggers

p-wassi opened the pull request:
The default LED triggers for netdev LAN and WAN got lost when switching
to ath79.

A general question for ar71xx->ath79: the boardname of this device has changed.
It was `gl-ar150` on ar71xx. On ath79 it is `glinet,ar150`. The 01_leds script breaks this
down to just `ar150`, the LEDs however are still called `gl-ar150`.
When I compare to the TP-Link devices: some are called `tplink,tl-xxx` and others
are just `tplink,xxx`. Should the vendor prefix be given in the second part of the boardname
or not?

p-wassi committed:
ath79: fix boardname of GL.iNet GL-AR150

This device is called GL-AR150, therefore rename the board
to 'gl-ar150'

Signed-off-by: Paul Wassi <p.wassi@gmx.at>

mkresin commented:
According to the [website](https://www.gl-inet.com/products/gl-ar150/), the name of the product is GL-AR150 and we should use this name. Considering the led names, the changed userspace boardname looks like a mistake and should be fixed.

p-wassi committed:
ath79: Add GL.iNet AR150 LED triggers

When switching from ar71xx to ath79 the default netdev LED
triggers for LAN and WAN got lost.

Signed-off-by: Paul Wassi <p.wassi@gmx.at>

p-wassi commented:
Thanks for checking, the GL-AR150 is now renamed.
(And the LED triggers added back)

Will do the same for ar300m -> gl-ar300m in a moment...

mkresin commented:
merged, thanks!
